gui/src/components/s3/account-management.vue

```diff
@@ -468,6 +477,122 @@
         </cortx-has-access>
       </v-col>
     </v-row>
+    
+    <v-dialog 
+      v-model="showResetAccountForm" 
+      persistent
+      max-width="500" 
```

Comment:
500 by default mean px. we should not be using px at all. Use em

Reply:
We can not add unit here, it works as an inline style for an element. Even if we add 'px' or 'em' it takes 'px' only

Reply:
Check with @shrimetta . If now PX can not be used, then we need to find some other way to use em or understand it wider impact

Reply:
To avoid 'PX' need to add external style using classes. But in this case 'dialog' component used at 2 places and both needs different width.